src/blocks/icon/svgs.js

```diff
@@ -1596,7 +1596,7 @@ let icons = bundledIconsEnabled ? {
 			__( 'sitemap', 'coblocks' ),
 		],
 		icon: <SVG xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><Path d="M0 0h24v24H0z" fill="none" /><Path d="M22 11V3h-7v3H9V3H2v8h7V8h2v10h4v3h7v-8h-7v3h-2V8h2v3z" /></SVG>,
-		icon_outlined: <SVG xmlns="http://www.w3.org/2000/svg" enable-background="new 0 0 24 24" viewBox="0 0 24 24"><Rect fill="none" height="24" width="24" /><Path d="M22,11V3h-7v3H9V3H2v8h7V8h2v10h4v3h7v-8h-7v3h-2V8h2v3H22z M7,9H4V5h3V9z M17,15h3v4h-3V15z M17,5h3v4h-3V5z" /></SVG>,
+		icon_outlined: <SVG xmlns="http://www.w3.org/2000/svg" enableBackground="new 0 0 24 24" viewBox="0 0 24 24"><Rect fill="none" height="24" width="24" /><Path d="M22,11V3h-7v3H9V3H2v8h7V8h2v10h4v3h7v-8h-7v3h-2V8h2v3H22z M7,9H4V5h3V9z M17,15h3v4h-3V15z M17,5h3v4h-3V5z" /></SVG>,
```

Comment:
Should these be a part of @godaddy-wordpress/coblocks-icons?

Reply:
They probably should be. That change is out of the scope of this PR but should be a story on the board for sure.